frame/system/src/lib.rs

```diff
@@ -278,7 +278,8 @@ pub mod pallet {
 			+ Debug
 			+ MaybeDisplay
 			+ Ord
-			+ MaxEncodedLen;
+			+ MaxEncodedLen
+			+ Clone;
```

Comment:
This is unnecessary, `Parameter` already implies `Clone`: https://paritytech.github.io/substrate/master/frame_support/dispatch/trait.Parameter.html

Reply:
Yeah, this didn't make much sense, other than that does the logic seem ok?